### PR TITLE
Predicates stuff 

### PR DESCRIPTION
**Pull request by savitharaghunathan**
For DMR, I have only added PVC & VSC.. Let me know if I need to edit that list

### CODE REVIEW

File: controllers/predicate.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		// Delete returns true if the Delete event should be processed
+		DeleteFunc: func(e event.DeleteEvent) bool {
+			return !e.DeleteStateUnknown && isObjectOursBackup(scheme, e.Object)
+		},
+	}
+}
+
+// isObjectOurs returns true if the object is ours.
+// it first checks if the object has our group, version, and kind
+// else it will check for non empty OadpOperatorlabel labels
```
Comment (shubham-pampattiwar) [inline controllers/predicate.go:33]: NIT: `// else it will check for non empty VolumeSnapshotMoverController labels`

Comment (savitharaghunathan) [reply controllers/predicate.go:33]: good catch..copy paste error :D

File: controllers/datamoverbackup_controller.go [hunk 2]
```diff
@@ -150,5 +153,10 @@ func (r *DataMoverBackupReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 func (r *DataMoverBackupReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&pvcv1alpha1.DataMoverBackup{}).
+		Owns(&snapv1.VolumeSnapshotContent{}).
+		Owns(&snapv1.VolumeSnapshot{}).
+		Owns(&v1.PersistentVolumeClaim{}).
+		Owns(&v1.Pod{}).
```
Comment (eemcmullan) [inline controllers/datamoverbackup_controller.go:159]: Should we own or watch the secret that we create? Same with ReplicationSource.

Comment (savitharaghunathan) [reply controllers/datamoverbackup_controller.go:159]: I thought we decided to not to add secrets for now. Correct me if I got that wrong from the sync up today

Comment (eemcmullan) [reply controllers/datamoverbackup_controller.go:159]: Yes you're correct. Sorry!

**Review by eemcmullan — APPROVED** [review]

**Pull request merged**